applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c

```diff
@@ -34,3 +34,84 @@ bool le_audio_ep_state_check(struct bt_bap_ep *ep, enum bt_bap_ep_state state)
 
 	return false;
 }
+
+int le_audio_freq_hz_get(const struct bt_audio_codec_cfg *codec, int *freq_hz)
+{
+	int ret;
+
+	ret = bt_audio_codec_cfg_get_freq(codec);
+	if (ret < 0) {
+		*freq_hz = 0;
+		return ret;
+	}
+
+	ret = bt_audio_codec_cfg_freq_to_freq_hz(ret);
+	if (ret < 0) {
+		*freq_hz = 0;
+		return ret;
+	}
+
+	*freq_hz = ret;
+
+	return 0;
+}
+
+int le_audio_duration_us_get(const struct bt_audio_codec_cfg *codec, int *frame_dur_us)
+{
+	int ret;
+
+	ret = bt_audio_codec_cfg_get_frame_dur(codec);
+	if (ret < 0) {
+		*frame_dur_us = 0;
+		return ret;
+	}
+
+	ret = bt_audio_codec_cfg_frame_dur_to_frame_dur_us(ret);
+	if (ret < 0) {
+		*frame_dur_us = 0;
+		return ret;
+	}
+
+	*frame_dur_us = ret;
+
+	return 0;
+}
+
+int le_audio_octets_per_sdu_get(const struct bt_audio_codec_cfg *codec, uint32_t *octets_per_sdu)
+{
+	int ret;
+
+	ret = bt_audio_codec_cfg_get_octets_per_frame(codec);
+	if (ret < 0) {
+		*octets_per_sdu = 0;
+		return ret;
+	}
+
+	*octets_per_sdu = ret;
+
+	if (*octets_per_sdu < LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE_MIN)) {
```

Comment:
The purpose of this function is to get the value and not check if it is valid.
In broadcast_sink this check is still performed. 
Remove line 92-98 and add the check back in unicast_server.